ledger-service/http-json-perf/src/main/scala/com/daml/http/perf/Main.scala

```diff
@@ -156,16 +156,15 @@ object Main extends StrictLogging {
               ledgerId.unwrap,
               ledgerPort,
               jsonApiJdbcConfig,
-              None,
             ) { (uri, _, _, ledger) =>
               for {
                 alicePartyData <- allocateUserAndJwt(ledger, "Alice")
                 bobPartyData <- allocateUserAndJwt(ledger, "Bob")
                 charliePartyData <- allocateUserAndJwt(ledger, "Charlie")
                 (exitCode, path) <- runGatlingScenario(
                   config,
-                  uri.authority.host.address,
-                  uri.authority.port,
```

Comment:
Now the `uri` arg is a `java.net.URI`, rather than an `akka.http.scaladsl.model.Uri`, so use switch to the appropriate methods.